src/position.js

```diff
@@ -5,7 +5,7 @@ const { ParsingError, InputValidationError } = require('./error');
 const { AA_PATTERN, AA_CODES } = require('./constants');
 
 
-const CDS_PATT = /(\d+)?([-+]\d+)?/;
+const CDS_PATT = /(-?\d+)?([-+]\d+)?/;
```

Comment:
what's the change in input text this allows for?

Reply:
See related ticket #5. We are adding support for negative pos values